SavedInstances/SavedInstances.lua

```diff
@@ -2142,8 +2151,36 @@ local function ShowIndicatorTooltip(cell, arg, ...)
     indicatortip:SetCell(n, 2, info.ID, "RIGHT", 2)
   end
   if info.Link then
+    local link = info.Link
+    if thisinstance.LFDID == 1944 then
+      -- Battle of Dazar'alor
+      -- https://github.com/SavedInstances/SavedInstances/issues/233
+      local locFaction = UnitFactionGroup("player")
+      if db.Toons[toon].Faction ~= locFaction then
+        local bits = tonumber(link:match(":(%d+)\124h"))
+        if db.Toons[toon].Faction == "Alliance" then
+          bits = bit.band(bits, 0x3134D)
+          if bit.band(bits, 0x1) > 0 then -- Grong the Revenant (Alliance)
+            bits = bit.bor(bits, 0x2)
+          end
+          if bit.band(bits, 0x4) > 0 then -- Jadefire Masters (Alliance)
+            bits = bit.bor(bits, 0x10)
+          end
+        else
+          bits = bit.band(bits, 0x3135A)
+          if bit.band(bits, 0x2) > 0 then -- Grong, the Jungle Lord (Horde)
+            bits = bit.bor(bits, 0x1)
+          end
+          if bit.band(bits, 0x10) > 0 then -- Jadefire Masters (Horde)
+            bits = bit.bor(bits, 0x4)
+          end
+        end
+        link = "\124cffff8000\124Hinstancelock:Player-0000-00000000:2070:"
+          .. diff .. ":" .. bits .. "\124h[Battle of Dazar'alor]\124h\124r"
```

Comment:
Isn't that going to always print the English name, even if the user's client is set to a different locale?

Reply:
No, `[Battle of Dazar'alor]` is the display name when it is sent in the chat, but SI set this link to a tooltip and get text from it. The text from tooltip is always based on local locale. I have tested this on zh-CN client (which I use), and it works perfectly.